controller/wingsail/controllers.py

```diff
@@ -0,0 +1,75 @@
+from controller.common.constants import CHORD_WIDTH_MAIN_SAIL, KINEMATIC_VISCOSITY
+from controller.common.lut import LUT
+
+
+class WingsailController:
+    def __init__(self):
+        self.lut = LUT()  # Placeholder for the LUT class
```

Comment:
When you use the LUT class, review the usage of `LUT` in your functions to make sure it's correct